R/pkg/R/SQLContext.R

```diff
@@ -149,6 +149,26 @@ createDataFrame <- function(sqlContext, data, schema = NULL, samplingRatio = 1.0
   dataFrame(sdf)
 }
 
+#' Create a DataFrame
+#'
+#' Converts R data.frame or list into DataFrame.
+#'
+#' @param sqlContext A SQLContext
+#' @param data An RDD or list or data.frame
+#' @param schema a list of column names or named list (StructType), optional
+#' @return an DataFrame
+#' @export
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' sqlContext <- sparkRSQL.init(sc)
+#' df1 <- as.DataFrame(sqlContext, iris)
+#' df2 <- as.DataFrame(sqlContext, list(3,4,5,6))
+#' }
+as.DataFrame <- function(sqlContext, data, schema = NULL, samplingRatio = 1.0){
```

Comment:
sqlContext is already a singleton, it's safe to call sparkRSQL.init multiple times. So we have move `sqlContext` to the second place (or even the last), and make it optional.